requirements.txt

```diff
@@ -3,3 +3,4 @@
 astropy
 matplotlib
 numpy
+katbeam
```

Comment:
Also - your commit is failing the unit tests because katbeam isn't in the `base_requirements.txt`.

Better here to install katbeam directly from github. Change the line to:

`katbeam @ git+https://github.com/ska-sa/katbeam`

Reply:
fixed also